MQTicketService/startApp.sh

```diff
@@ -20,4 +20,14 @@ sleep 30
 # Set environment
 . /opt/mqm/bin/setmqenv -s
 # Execute TicketGenerator
-/jdk-11.0.13+8-jre/bin/java -cp /data/TicketGenerator/target/TicketGenerator-1.4.jar com.ibm.mq.badge.Manager
\ No newline at end of file
+
+if [ "$envPlatformArch" == "amd64" ]
+then
+    /jdk-11.0.13+8-jre/bin/java -cp /data/TicketGenerator/target/TicketGenerator-1.4.jar com.ibm.mq.badge.Manager 
+elif [ "$envPlatformArch" == "arm64" ] 
+then
+    /jdk-11.0.22+7-jre/bin/java -cp /data/TicketGenerator/target/TicketGenerator-1.4.jar com.ibm.mq.badge.Manager 
+ else 
+    echo "Error: Valid platform architectures are amd64 or arm64" 
+    exit 1
```

Comment:
Was this tested to see what happens if neither `amd64` or `arm64` have been set.

We are running a command inside a docker build. The run will exit, but not the docker build